src/network/transaction_request/mod.rs

```diff
@@ -9,7 +9,8 @@ use crate::network::{tx_type::TxType, unsigned_tx::eip712::TxEip712};
 use super::unsigned_tx::eip712::{hash_bytecode, BytecodeHashError, PaymasterParams};
 use super::{unsigned_tx::eip712::Eip712Meta, Zksync};
 
-#[derive(Debug, Default, Clone, serde::Serialize, serde::Deserialize)]
+/// Transaction request supporting ZKsync's EIP-712 transaction types.
```

Comment:
```suggestion
/// Transaction request supporting ZKsync's EIP-712 transaction types.
///
/// Unlike `TransactionRequest` for Ethereum network, it would try to use ZKsync-native
/// EIP712 transaction type by default, unless explicitly overridden. The reason for that
/// is that EIP712 transactions have the same capabilities as type 0 and EIP1559
/// transactions, while being cheaper to process.
```